src/components/Editor/Repeat/RepeatSummary.vue

```diff
@@ -21,14 +21,270 @@
+						dayOfMonthList
+					})
+				} else {
+					const ordinalNumber = this.getTranslatedOrdinalNumber(this.bySetPosition)
+					const byDaySet = this.getTranslatedByDaySet(this.byDay)
+
+					limitPart = this.$t('calendar', 'on the {ordinalNumber} {byDaySet}', {
+						ordinalNumber,
+						byDaySet
+					})
+				}
+			} else if (this.frequency === 'YEARLY') {
+				const monthNames = this.getTranslatedMonths(this.byMonth)
+
+				if (this.byDay.length === 0) {
+					limitPart = this.$t('calendar', 'in {monthNames}', {
+						monthNames
+					})
+				} else {
+					const ordinalNumber = this.getTranslatedOrdinalNumber(this.bySetPosition)
+					const byDaySet = this.getTranslatedByDaySet(this.byDay)
+
+					limitPart = this.$t('calendar', 'in {monthNames} on the {ordinalNumber} {byDaySet}', {
+						monthNames,
+						ordinalNumber,
+						byDaySet
+					})
+				}
+			}
+
+			let endPart = ''
+			if (this.until !== null) {
+				let untilDate = moment(this.until).format('L')
+
+				endPart = this.$t('calendar', ', until {untilDate}', {
+					untilDate
+				})
+			} else if (this.count !== null) {
+				endPart = this.$n('calendar', ', %n time', ', %n times', this.count)
+			}
+
+			return [
+				freqPart,
+				limitPart,
+				endPart
+			].join(' ').trim()
+		}
+	},
+	methods: {
+		/**
+		 * Gets the translated ordinal number for by-set-position
+		 *
+		 * @param {Number} bySetPositionNum The by-set-position number to get the translation of
+		 * @returns {string}
+		 */
+		getTranslatedOrdinalNumber(bySetPositionNum) {
+			switch (bySetPositionNum) {
+			case 1:
+				return this.$t('calendar', 'first')
+
+			case 2:
+				return this.$t('calendar', 'second')
+
+			case 3:
+				return this.$t('calendar', 'third')
+
+			case 4:
+				return this.$t('calendar', 'fourth')
+
+			case 5:
+				return this.$t('calendar', 'fifth')
+
+			case -2:
+				return this.$t('calendar', 'second to last')
+
+			case -1:
+				return this.$t('calendar', 'last')
+
+			default:
+				return ''
+			}
+		},
+		/**
+		 * Gets the byDay list as formatted list of translated weekdays
+		 *
+		 * @param {string[]} byDayList The by-day-list to get formatted
+		 * @returns {string}
+		 */
+		getTranslatedByDaySet(byDayList) {
+			const byDayNames = []
+			const allByDayNames = getDayNames()
+
+			if (byDayList.includes('MO')) {
+				byDayNames.push(allByDayNames[1])
+			}
+			if (byDayList.includes('TU')) {
+				byDayNames.push(allByDayNames[2])
+			}
+			if (byDayList.includes('WE')) {
+				byDayNames.push(allByDayNames[3])
+			}
+			if (byDayList.includes('TH')) {
+				byDayNames.push(allByDayNames[4])
+			}
+			if (byDayList.includes('FR')) {
+				byDayNames.push(allByDayNames[5])
+			}
+			if (byDayList.includes('SA')) {
+				byDayNames.push(allByDayNames[6])
+			}
+			if (byDayList.includes('SU')) {
+				byDayNames.push(allByDayNames[0])
+			}
+
+			return byDayNames.join(', ')
```

Comment:
Support is pretty bad right now but we could use [`Intl.ListFormat.format()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat) here.

Reply:
https://caniuse.com/#feat=internationalization

Support seems not that bad anymore

Reply:
I feel like this caniuse page is generic and concerns things like `Intl.DateTimeFormat` (for which support is good).

Reply:
Ah, yes.
MDN shows way more red: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat#Browser_compatibility